docs/data/sdks/marketing-analytics-browser/migration.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Browser SDK 2.0 Migration Guide
+description: Use this guide to easily migrate from Amplitude Marketing Analytics Browser to the Browser SDK 2.0.
+---
+
+Amplitude Browser SDK 2.0 (`@amplitude/analytics-browser`) features default event tracking, improved marketing attribution tracking, simplified interface and a lighter weight package.
+
+### Terminology
+
+* `@amplitude/marketing-analytics-browser@1`: Marketing Analytics Browser SDK
+* `@amplitude/analytics-browser@2`:  Browser SDK 2.0
+
+### Dependency
+
+dependency.md
+
+```diff
+{
+  "dependencies": {
+-     "@amplitude/marketing-analytics-browser": "^1"
++     "@amplitude/analytics-browser": "^2"
+  }
+}
+```
+
+### Default events tracking
+
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/default-event-tracking.md"
+
+### Deprecates user agent client-side parsing
+
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/user-agent-parsing.md"
+
+### Marketing attribution tracking
+
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/marketing-attribution-tracking-1.md"
+
+Marketing Analytics Browser SDK by default, allows other subdomains to be tracked as referrer. If this is behavior is desired, refer to the code below.
+
+--8<-- "includes/sdk-migration/sdk-browser-2-migration/marketing-attribution-tracking-2.md"
+
+### Page View tracking
+
+#### Moved `options.pageViewTracking` to `options.defaultTracking.pageViews`
+
+```diff
+  amplitude.init(API_KEY, undefined, {
+-   pageViewTracking: {
+-     trackOn: true,
+-     trackHistoryChanges: 'pathOnly', 
+-     eventType: 'Page View',
++   defaultTracking: {
++     trackOn: true,
++     trackHistoryChanges: 'pathOnly', 
++     eventType:'Page View',
```

Comment:
please refer to page view default tracking here: https://github.com/amplitude/Amplitude-TypeScript/blob/main/packages/analytics-types/src/config/browser.ts#L34